core/src/main/java/com/exadel/etoolbox/rolloutmanager/core/servlets/CollectLiveCopiesServlet.java

```diff
@@ -139,13 +154,16 @@ private JsonObject relationshipToJson(LiveRelationship relationship,
 
         String liveCopyPath = liveCopy.getPath();
         boolean isNew = !resourceExists(resourceResolver, liveCopyPath + syncPath);
+
         return Json.createObjectBuilder()
                 .add(MASTER_JSON_FIELD, source + sourceSyncPath)
                 .add(PATH_JSON_FIELD, liveCopyPath + syncPath)
                 .add(DEPTH_JSON_FIELD, depth)
                 .add(LIVE_COPIES_JSON_FIELD, getLiveCopiesJsonArray(liveCopyPath, syncPath, resourceResolver, depth + 1))
                 .add(IS_NEW_JSON_FIELD, isNew)
                 .add(HAS_ROLLOUT_TRIGGER_JSON_FIELD, !isNew && hasAutoTrigger(liveCopy))
+                .add(LAST_ROLLED_OUT_TIME_AGO_JSON_FIELD, getRolledOutAgoJsonField(resourceResolver, liveCopyPath + syncPath))
```

Comment:
Since I had a little more options on the backend side and the code looked more aesthetically pleasing, I've implemented it on the backend. In general, I agree, logically and to support i18n it is better to do it on the frontend. I implemented, and it  turned out not as bad as I expected. Thank you!